third_party/llvm-project/libcxx/src/thread.cpp

```diff
@@ -75,7 +75,8 @@ thread::detach()
 unsigned
 thread::hardware_concurrency() noexcept
 {
-#if defined(_SC_NPROCESSORS_ONLN)
+// Excluded from Modular builds as Starboard doesn't provide sysconf
+#if defined(_SC_NPROCESSORS_ONLN) && !SB_IS(MODULAR)
```

Comment:
Why is this change necessary in this PR?

Reply:
The _SC_NPROCESSORS_ONLN is defined in <unistd.h> which triggers sysconf.
In this PR we start using <unistd.h> for `usleep`.